code/src/App.js

```diff
@@ -1,12 +1,34 @@
 import React from 'react'
 import data from './data.json'
+import { AlbumCard } from './components/AlbumCard'
+import { Header } from './components/Header'
 
+const albumArray = data.albums.items;
 console.log(data)
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    return (
+      <main>
+        <img class="spotifake-logo" src={require(`./components/icons/spotifake_logo.png`)}/>
+        
+        <Header />
+        <div className="album-grid">
+          {albumArray.map(album => (
+            <AlbumCard 
+            key={album.artists[0].id}
+            albumName={ album.name }
+            artistName={ album.artists[0].name }
+            image={ album.images[0].url }
+            albumURL={ album.external_urls.spotify }
+            artistURL={ album.artists[0].external_urls.spotify }
+            artists={ album.artists }
+            />  
+            
+          ))}
+
+        </div>
+      </main>
+    )
```

Comment:
> The things I feel I don't really get yet are how I'm supposed to work with scripting inside the returns – when to use curly brackets, the syntax to work with strings and variables, etc.

When you return something on several lines, we should wrap the return statement in regular parentheses. If we want to write JavaScript inside the return statement we need to specify that with curly brackets, just like you did around the `albumArray.map()` here. Hope that clear things up a bit :)

Reply:
PS. Get rid of the empty lines inside the `return`